### DISCUSSION TITLE
Getting 'No config named' when trying to load the config file

### DISCUSSION THREAD

**Discussion by laxman356** [strongswan/strongswan#1717] [Q&A]
We are running strongswan version 5.2.1 which is very old compared to the latest 5.9.x. We are frequently seeing 'No config named' when I try to bounce the tunnel even when the we have the right config.
The same config was running earlier but it is failing now with the error  'No config named'.
I do not see any logs. The service is running fine. This behavior is passing on to the other tunnel configs as well.

What could be the reason?

**Comment by tobiasbrunner** [comment]
> We are running strongswan version 5.2.1 which is very old compared to the latest 5.9.x.

It is and I wouldn't recommend doing that.

> We are frequently seeing 'No config named' when I try to bounce the tunnel even when the we have the right config.

What does "bounce the tunnel" mean?

> What could be the reason?

No idea. Check the output of `ipsec statusall` to see if you actually have a config loaded with a particular name.

**Comment by laxman356** [comment]
Hi Tobias,
We are using ipsec cli for our config.
We have a tunnel config for example test.tun

This test.tun which was UP earlier suddenly stopped working.
Bounce the tunnel means bringing the existing tunnel down and then bringing it back up. I ran these two commands

**sudo ipsec down test**
this brought down the existing tunnel and then ran 
**sudo ipsec up test** to bring the tunnel back up.

When I executed **sudo ipsec up test** this gave me the error saying 'No config named test' even when the config existed.

ipsec couldn't read the tunnel config.
No changes were made to the config.  We have around over 100 tunnels and it is the same case for all of them.

We restarted strongswan service but still it was showing the same error.
Later we killed charon and ipsec process and then did sudo systemctl start strongswan, which loaded all the configs, strongswan was able to read the tunnel configs and all tunnels came up successfully.

**Reply by tobiasbrunner** [reply]
> When I executed **sudo ipsec up test** this gave me the error saying 'No config named test' even when the config existed.

Are you sure? What does `ipsec statusall` say?

> ipsec couldn't read the tunnel config.

What do you mean?
